src/db_common.rs

```diff
@@ -19,4 +19,16 @@ impl DbInner {
             .send(FlushThreadMsg::Flush(None))?;
         Ok(())
     }
+
+    pub(crate) fn maybe_freeze_wal(
+        &self,
+        guard: &mut RwLockWriteGuard<'_, DbState>,
+    ) -> Result<(), SlateDBError> {
+        if guard.wal().size() < 64 * 1024 * 1024 {
```

Comment:
let's make sure to use the same value in the db options for L0 max size in bytes